src/Bicep.Core/Emit/TemplateWriter.cs

```diff
@@ -114,6 +114,11 @@ public void Write(SourceAwareJsonTextWriter writer)
                 emitter.EmitProperty("languageVersion", "1.10-experimental");
             }
 
+            if (Context.Settings.EnableAsserts)
+            {
+                emitter.EmitProperty("languageVersion", "1.11-experimental");
+            }
```

Comment:
the `1.11-experimental` in Deployments actually has everything the `1.10-experimental` has plus the Asserts. I could do a check on both EnableSymbolicNames and EnableAsserts to avoid emitting 2 `languageVersion` properties if the 1.11 needs to be kept, but I'm not sure

Reply:
Right, but can we just add asserts to `1.10-experimental` and remove `1.11-experimental` from ARM? I don't see the need for a new language version here.

Reply:
I just updated this PR to match the latest change in language version from your PR in Deployments